CONTRIBUTING.md

```diff
@@ -36,3 +36,15 @@ to work on. You can also filter the issues via label:
   Might require some experience.
 * [Help wanted](https://github.com/nsxiv/nsxiv/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22):
   (Intermediate/Experienced) Issues where we require some help.
+
+Development workflow
+--------------------
+
+Our workflow regarding pull requests (PR) is:
```

Comment:
This makes it sound like it's applicable to anyone sending a PR when reality it's only for maintainers.